bigbluebutton-html5/imports/api/annotations/server/handlers/whiteboardSend.js

```diff
@@ -1,7 +1,24 @@
 import { check } from 'meteor/check';
-
+import { AnnotationsStreamer } from '/imports/api/annotations';
 import addAnnotation from '../modifiers/addAnnotation';
 
+let annotationsQueue = {};
+let annotationsRecieverIsRunning = false;
+
+const proccess = () => {
+  if (!Object.keys(annotationsQueue).length) {
+    annotationsRecieverIsRunning = false;
+    return;
+  }
+  annotationsRecieverIsRunning = true;
+  Object.keys(annotationsQueue).forEach(meetingId => {
+    AnnotationsStreamer.emit('added', { meetingId, annotations: annotationsQueue[meetingId] });
+  });
+  annotationsQueue = {};
+
+  Meteor.setTimeout(proccess, 60);
```

Comment:
Is this 60ms worth putting in a config or a const?

Reply:
Not worth. Kappa.

Reply:
jk, will move that to a const dont think wee need a config file for that.